realtor-service/src/main/resources/application.properties

```diff
@@ -1,2 +1,2 @@
 spring.application.name=realtor-service
-feign.hystrix.enabled=true
+feign.hystrix.enabled=true
```

Comment:
I guess, to separate different types of configuration and keep clean code, but lets @banadiga answer this.